src/WebJobs.Extensions.DocumentDB/Trigger/CosmosDBTriggerAttributeBindingProvider.cs

```diff
@@ -144,5 +160,55 @@ private bool TryReadFromSettings(string settingsKey, string defaultValue, out st
             settingsValue = _nameResolver.Resolve(settingsKey);
             return !string.IsNullOrEmpty(settingsValue);
         }
+
+        private static async Task CreateIfNotExistAsync(IDocumentDBService service, string databaseName, string collectionName, int throughput)
+        {
+            await DocumentDBUtility.RetryAsync(() => CreateDatabaseIfNotExistsAsync(service, databaseName),
+                DefaultMaxThrottleRetries, codesToIgnore: HttpStatusCode.Conflict);
```

Comment:
Yeah, I'm removing the Utility from here, the SDK should handle retries internally (since version 1.8.0). Also I'm adding two methods to the `IDocumentDBService`: `CreateDatabaseIfNotExistsAsync` and `CreateDocumentCollectionIfNotExistsAsync` which are internally in the client.